src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -160,22 +160,56 @@ struct CopyOp<DstViewType, SrcViewType, 3>
 };
 } // namespace PermuteHelper
 
-template <typename ExecutionSpace, typename PermutationView, typename View>
+template <typename ExecutionSpace, typename PermutationView, typename InputView,
+          typename OutputView>
+void applyReversePermutation(ExecutionSpace const &space,
```

Comment:
While googling, realized that it is actually called *inverse* permutation, not reverse.